### PR TITLE
Refer to the Wildfly distribution in the operator guides

### LINKED ISSUE #292: Clean "Getting started" guides regarding the operator
### Description

Review and update or remove the "Getting started" guides for OpenShift and Kubernetes using the operator.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andreaTP**
Resolves #292 

This is the result:
Screenshot 2022-04-14 at 12 08 09
and:
Screenshot 2022-04-14 at 12 08 18

### CODE REVIEW

File: guides/getting-started/getting-started-operator-kubernetes.adoc [hunk 1]
```diff
@@ -1,5 +1,9 @@
 :guide-title: Kubernetes Operator
-:guide-summary: Get started with Keycloak Operator on Kubernetes
+:guide-summary: Get started with Keycloak Operator on Kubernetes [Wildfly distribution]
+
+.Note:
+[NOTE]
+This guide refers to the Wildfly distribution of Keycloak.
```
Comment (vmuzikar) [inline guides/getting-started/getting-started-operator-kubernetes.adoc:6]: I think we should be even more explicit. Something like:
```suggestion
This guide refers to the https://github.com/keycloak/keycloak-operator[old operator] and Wildfly distribution of Keycloak.
```
WDYT?

File: guides/getting-started/getting-started-operator-openshift.adoc [hunk 2]
```diff
@@ -1,5 +1,9 @@
 :guide-title: OpenShift Operator
-:guide-summary: Get started with Keycloak Operator on Openshift
+:guide-summary: Get started with Keycloak Operator on Openshift [Wildfly distribution]
+
+.Note:
+[NOTE]
+This guide refers to the Wildfly distribution of Keycloak.
```
Comment (vmuzikar) [inline guides/getting-started/getting-started-operator-openshift.adoc:6]:
```suggestion
This guide refers to the https://github.com/keycloak/keycloak-operator[old operator] and Wildfly distribution of Keycloak.
```

**Review by vmuzikar — CHANGES_REQUESTED** [review]

File: guides/getting-started/getting-started-operator-kubernetes.adoc (hunk 1 above)
Comment (DGuhr) [reply guides/getting-started/getting-started-operator-kubernetes.adoc:6]: i'd say yes, and also perhaps add "legacy" somewhere, and point people to the latest greatest one? e.g. 
```
This guide refers to the https://github.com/keycloak/keycloak-operator[old, unmaintained operator] and legacy Wildfly distribution of Keycloak. The guide for the new operator can be found https://...[here].

```
?

File: guides/getting-started/getting-started-operator-openshift.adoc (hunk 2 above)
Comment (DGuhr) [reply guides/getting-started/getting-started-operator-openshift.adoc:6]: see above ;)

File: guides/getting-started/getting-started-operator-openshift.adoc [hunk 3]
```diff
@@ -1,5 +1,9 @@
 :guide-title: OpenShift Operator
-:guide-summary: Get started with Keycloak Operator on Openshift
+:guide-summary: Get started with Keycloak Operator on Openshift [Wildfly distribution]
```
Comment (DGuhr) [inline guides/getting-started/getting-started-operator-openshift.adoc:2]:
```suggestion
:guide-summary: Get started with Keycloak Operator on Openshift [Legacy Wildfly]
```

Comment (DGuhr) [reply guides/getting-started/getting-started-operator-openshift.adoc:2]: can we add labels right to the headline? ;)

File: guides/getting-started/getting-started-operator-kubernetes.adoc [hunk 4]
```diff
@@ -1,5 +1,9 @@
 :guide-title: Kubernetes Operator
-:guide-summary: Get started with Keycloak Operator on Kubernetes
+:guide-summary: Get started with Keycloak Operator on Kubernetes [Wildfly distribution]
```
Comment (DGuhr) [inline guides/getting-started/getting-started-operator-kubernetes.adoc:2]:
```suggestion
:guide-summary: Get started with Keycloak Operator on Kubernetes [Legacy Wildfly]
```

File: guides/getting-started/getting-started-operator-kubernetes.adoc (hunk 1 above)
Comment (vmuzikar) [reply guides/getting-started/getting-started-operator-kubernetes.adoc:6]: I would not use "unmaintained" wording. :) It is maintained, just deprecated.

Comment (DGuhr) [reply guides/getting-started/getting-started-operator-kubernetes.adoc:6]: right :) deprecated is better. that was just what came to my mind

**Review by vmuzikar — APPROVED** [review]

**Review by DGuhr — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**